devices.js

```diff
@@ -1882,6 +1910,16 @@ const devices = [
         fromZigbee: [fz.heiman_water_leak],
         toZigbee: [],
     },
+    
+    {
+        zigbeeModel: ['HOMA2023'],
+        model: 'ZB-CC-40',
```

Comment:
Could you provide a link to this device (to be added to the documentation)?

Reply:
its a noname LED driver from Aliexpress.. look like a Gledopto driver.
one of our users has bought it

Reply:
Can you give the aliexpress link?